src/acceleration/IQNILSAcceleration.cpp

```diff
@@ -52,7 +53,7 @@ void IQNILSAcceleration::updateDifferenceMatrices(
   BaseQNAcceleration::updateDifferenceMatrices(cplData);
 }
 
-void IQNILSAcceleration::computeQNUpdate(const DataMap &cplData, Eigen::VectorXd &xUpdate)
+void IQNILSAcceleration::computeQNUpdate(Eigen::VectorXd &xUpdate)
```

Comment:
Maybe directly merge some of these refactorings into `develop` to make this PR here a bit less  crowded?

Pulling out the refactorings is a bit tedious but I think it really helps us to concentrate on the important parts here. We can also do this together on Thursday.